web/css/index.css

```diff
@@ -477,7 +477,7 @@ header.active {
 
 #help-links {
   height: 6rem;
-  background-color: var(--color-help);
+  background-color: #e7e5e2;
```

Comment:
We now have a css variable for this color called `--color-sentence`. Let's change the name of that to `--color-secondary`, and use that variable here and for the sentence-box in the css file. Does that make sense?